src/Money.php

```diff
@@ -515,6 +515,66 @@ public function jsonSerialize()
         ];
     }
 
+    /**
+     * @param Money   $first
+     * @param Money[] $collection
```

Comment:
This isn't correct. The second argument is not an array. I believe I saw `...Money` notation somewhere, but I'm not sure.